index.html

```diff
@@ -1,42 +1,91 @@
 <!DOCTYPE html>
 <html lang="en">
-  <head>
-    <meta charset="UTF-8" />
-    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Document</title>
-    <!-- Bootstrap Styles-->
-    <link
-      rel="stylesheet"
-      href="https://stackpath.bootstrapcdn.com/bootstrap/4.4.1/css/bootstrap.min.css"
-      integrity="sha384-Vkoo8x4CGsO3+Hhxv8T/Q5PaXtkKtu6ug5TOeNV6gBiFeWPGFN9MuhOf23Q9Ifjh"
-      crossorigin="anonymous"
-    />
-
-    <!-- INCLUDE STYLES FILE HERE -->
-  </head>
-
-  <body>
-    <div class="container">
-      <!-- WORK FROM INSIDE THIS CONTAINER-->
+
+
+<head>
+  <meta charset="UTF-8" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <title> Parneet Kang</title>
+  <!-- Bootstrap Styles-->
+  <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.4.1/css/bootstrap.min.css"
+    integrity="sha384-Vkoo8x4CGsO3+Hhxv8T/Q5PaXtkKtu6ug5TOeNV6gBiFeWPGFN9MuhOf23Q9Ifjh" crossorigin="anonymous" />
+
+  <!-- INCLUDE STYLES FILE HERE -->
+  <link rel="stylesheet" href="resources/styles/styles.css" />
+</head>
+
+<body>
+  <div class="container">
+    <!-- WORK FROM INSIDE THIS CONTAINER-->
+    <h1> Parneet Kang</h1>
+
+    <img src="resources/images/Website.png" />
```

Comment:
Try to always set the alt tag for your images. The alt attribute holds a text description of the image, which isn't mandatory but is incredibly useful for accessibility — screen readers read this description out to visually impaired users. Including alt text with your images ensures all users, regardless of visual ability, can appreciate the content on your site.